src/D2L.Bmx/Program.cs

```diff
@@ -21,8 +23,28 @@
 // allow no-sandbox argument for chromium to for passwordless auth with elevated permissions
 var bypassBrowserSecurityOption = new Option<bool>(
 	name: "--experimental-bypass-browser-security",
+	getDefaultValue: () => false,
 	description: ParameterDescriptions.ExperimentalBypassBrowserSecurity );
 
+bypassBrowserSecurityOption.AddValidator( result => {
+	bool bypass = result.GetValueForOption( bypassBrowserSecurityOption );
+	bool isElevated = false;
+	if( OperatingSystem.IsWindows() ) {
+		isElevated = new WindowsPrincipal( WindowsIdentity.GetCurrent() ).IsInRole( WindowsBuiltInRole.Administrator );
+	} else if( OperatingSystem.IsLinux() || OperatingSystem.IsMacOS() ) {
+		isElevated = NativeMethods.GetPosixEuid() == 0;
+	}
+	if( isElevated && !bypass ) {
+		result.ErrorMessage = """
+			Running BMX with elevated permissions is only allowed with the '--experimental-bypass-browser-security' flag.
+			Only include this if you aren't concerned with the security of the host for your Okta organization.
```

Comment:
nit: I would really like it if we could articulate that:
1. Using BMX to do AWS stuff is perfectly  fine while elevated
2. You only need this flag when you want to do the equivalent of ``bmx login`` as admin
    a. Right call in most situations is to, run ``bmx login`` on your computer once a day without elevated privs
    b. If you insist in running BMX with elevated privs as the first thing of the day, that is when you need the flag.
    
 Not sure how to fit all that in one sentence :P

Reply:
Okay so this checking for permission logic should go in the okta authenticator class then

Reply:
There's also the consideration that... what if this flag is used with AWS credential process, and what if that may be triggered automatically sometimes elevated sometimes not?
This is a real concern for the LMS.

Reply:
in other words, I'm not too sure that we should always fail BMX when this flag is present but the user isn't elevated

Reply:
Yup I'll just move it to the okta authenticator and divert to password if it's not provided and they are admin

Reply:
Okay I moved it to the authenticator where if bmx is running as admin and no flag provided, we fall back to password. If not admin but flag provided, we set the flag to false so '--no-sandbox' doesn't get included

Reply:
I think that should also address the issue with wording you had Adipa since it's only on attempting to login now